src/include/common/typedefs.h

```diff
@@ -214,6 +214,7 @@ using byte = std::byte;
 STRONG_TYPEDEF(timestamp_t, uint64_t);
 STRONG_TYPEDEF(layout_version_t, uint32_t);
 STRONG_TYPEDEF(col_id_t, uint16_t);
+STRONG_TYPEDEF(col_oid_t, uint32_t);
```

Comment:
Not in your PR, but this `col_id_t`, `col_oid_t` thing is making me sad.

I will probably later refactor all these strong typedefs into namespaces. (that way we will get `storage::col_id_t` and `catalog::col_oid_t`)

Reply:
That sounds reasonable.

Reply:
Want me to add a todo in my PR?